packages/ionic/src/lib/project/angular/serve.ts

```diff
@@ -85,9 +102,16 @@ The dev server can use HTTPS via the ${chalk.green('--ssl')} option ${chalk.bold
     const ssl = options['ssl'] ? Boolean(options['ssl']) : undefined;
     const configuration = options['configuration'] ? String(options['configuration']) : (prod ? 'production' : undefined);
     const sourcemaps = typeof options['source-map'] === 'boolean' ? Boolean(options['source-map']) : undefined;
+    const consolelogs = typeof options['consolelogs'] === 'boolean' ? Boolean(options['consolelogs']) : undefined;
+    let consolelogsPort;
+    if (consolelogs) {
+      consolelogsPort = str2num(options['consolelogs-port'], DEFAULT_CONSOLE_LOGS_PORT);
```

Comment:
Why not just:

```ts
const consolelogsPort = consolelogs ? str2num(options['consolelogs-port'], DEFAULT_CONSOLE_LOGS_PORT) : undefined;
```

Reply:
Ah good call, that block went through several incarnations as I was testing out the various args combos and how it was printed by the ng cli.